engine/runtime/src/main/java/org/enso/interpreter/node/callable/InteropMethodCallNode.java

```diff
@@ -60,7 +59,8 @@ InvokeMethodNode buildSorter(int length) {
         args,
         InvokeCallableNode.DefaultsExecutionMode.EXECUTE,
         InvokeCallableNode.ArgumentsExecutionMode.PRE_EXECUTED,
-        0);
+        0,
+        true);
```

Comment:
`InteropMethodCallNode` does _interop call_ `onBoundary`. Thus passing `true` in.